opnode/rollup/driver/step.go

```diff
@@ -37,8 +39,8 @@ func (d *outputImpl) newBlock(ctx context.Context, l2Finalized eth.BlockID, l2Pa
 	}
 
 	timestamp := l2Info.Time() + d.Config.BlockTime
-	if timestamp >= l1Info.Time() {
-		return l2Parent, nil, errors.New("L2 Timestamp is too large")
+	if timestamp >= l1Info.Time()+d.Config.MaxSequencerTimeDiff {
+		return l2Parent, nil, newBlockTooNew
```

Comment:
Do we have any test coverage for this error case?

Reply:
The e2e test used to always hit this and then not recognize this error is expected behavior. Probably should have a more explicit test though, will try that